### PR TITLE
chore: save raw value for properties and forward this information

### PR DESCRIPTION
**Pull request by gabrielfs7**
https://oat-sa.atlassian.net/browse/ADF-1194

# Goal 

- Forward to FE sortable columns
- Forward to FE all the fields that can be displayed in the table.

# TODO

- [x] Finish unit tests
- [x] Validate with FE that the solution works
- [x] Make sure that with GenerisSearch (advanced search disabled) it will still work

### CODE REVIEW

File: config/index.conf.php [hunk 1]
```diff
@@ -20,343 +20,13 @@
...
-        ],
-    ],
+    'items' => require __DIR__ . '/items.conf.php',
+    'tests' => require __DIR__ . '/tests.conf.php',
+    'groups' => require __DIR__ . '/groups.conf.php',
+    'deliveries' => require __DIR__ . '/deliveries.conf.php',
+    'delivery-results' => require __DIR__ . '/delivery-results.conf.php',
+    'test-takers' => require __DIR__ . '/test-takers.conf.php',
+    'assets' => require __DIR__ . '/assets.conf.php',
+    'property-list' => require __DIR__ . '/property-list.conf.php',
```
Comment (hectoras) [inline config/index.conf.php:31]: nice :ok_hand:

File: config/test-takers.conf.php [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ *
+ * Copyright (c) 2022 (original work) Open Assessment Technologies SA;
+ */
+
+declare(strict_types=1);
+return [
```
Comment (hectoras) [inline config/test-takers.conf.php:22]: nitpick
```suggestion
declare(strict_types=1);

return [
```

File: config/tests.conf.php [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+            'dynamic_templates' => require __DIR__ . '/dynamic-templates.conf.php',
+        ],
+        'settings' => [
+            'index' => [
+                'number_of_shards' => '1',
+                'number_of_replicas' => '1',
+            ],
+        ],
+    ],
+];
```
Comment (hectoras) [inline config/tests.conf.php:58]: nitpick

```suggestion
];

```

**Review by hectoras — APPROVED** [review]
- [x] Pull request title and description are meaningful
- [x] Pull request's target is not `master`
- [x] Target branch is correct
- [x] Commits are following conventional commits
- [x] New code is covered by tests (if applicable)
- [x] New code is respecting code style rules
- [x] New code is respecting best practices
- [x] New code is not subject to concurrency issues (if applicable)
- [ ] Feature is working correctly on my local machine (if applicable)
- [ ] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [x] Acceptance criteria are respected
- [x] Kitchen links are not exposed in the description

**Review by shpran — APPROVED** [review]
- [x] New code is covered by tests (if applicable)
- [ ] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [ ] New code is respecting code style rules
- [x] New code is respecting best practices
- [x] New code is not subject to concurrency issues (if applicable)
- [ ] Feature is working correctly on my local machine (if applicable)
- [x] Acceptance criteria are respected
- [x] Pull request title and description are meaningful
- [x] Pull request's target is not `master`
- [x] Commits are following [conventional commits](https://www.conventionalcommits.org/en/v1.0.0/)
- [x] Commits messages are meaningful
- [x] Commits are atomic
- [ ] Changelog is updated according to changes (if applicable)
- [ ] Documentation is updated according to changes (if applicable)

File: config/test-takers.conf.php (hunk 2 above)
Comment (gabrielfs7) [reply config/test-takers.conf.php:22]: done

**Pull request merged**